src/cryptsetup/cryptsetup.c

```diff
@@ -220,6 +220,11 @@ static int parse_one_option(const char *option) {
                 arg_submit_from_crypt_cpus = true;
         else if (streq(option, "luks"))
                 arg_type = ANY_LUKS;
+        /* since cryptsetup 2.3.0 (Feb 2020) */
+        #ifdef CRYPT_BITLK
```

Comment:
please move the first column. we typically indent pre processor macros if they themselves are nested.